scripts/install/quassel.sh

```diff
@@ -24,29 +24,32 @@ distribution=$(lsb_release -is)
 codename=$(lsb_release -cs)
 IP=$(ip route get 1 | sed -n 's/^.*src \([0-9.]*\) .*$/\1/p')
 user=$(cut -d: -f1 < /root/.master.info)
-. /etc/swizzin/sources/functions/backports
 
 echo "Installing Quassel PPA (Ubuntu) or grabbing latest backport (Debian) ... "
 
 if [[ $distribution == Ubuntu ]]; then
   echo "Installing Quassel PPA"
-  apt-get install -q -y software-properties-common > /dev/null 2>&1
+  apt_install software-properties-common --skip-update
 	apt-add-repository ppa:mamarley/quassel -y >/dev/null 2>&1
-	apt-get -q -y update >/dev/null 2>&1
-  apt-get -q -y install quassel-core >/dev/null 2>&1
+  apt_update
+  apt_install quassel-core
 else
+  #shellcheck source=sources/functions/backports
+  . /etc/swizzin/sources/functions/backports
   if [[ $codename == "buster" ]]; then
     echo "Grabbing latest release"
-    apt-get -y -q install quassel-core > /dev/null 2>&1
+    apt_install quassel-core
   elif [[ $codename == "stretch" ]]; then
     check_debian_backports
     echo "Grabbing latest backport"
-    apt-get -y -q install quassel-core > /dev/null 2>&1
+    set_packages_to_backports quassel-core
```

Comment:
This should be checked as the diff doesn't say this was grabbed from backports before